config/main.go

```diff
@@ -5,7 +5,7 @@ var (
 	Port int    = 7379
 )
 
-var KeysLimit int = 100
+var KeysLimit int = 10000000
```

Comment:
Let's leave the default value for keys a smaller number. We can go with 10k for now.
As for the benchmarking, please set the config value as a part of the benchmark function and set it to `10000000` so that we don't modify the default global value just to make the benchmark run.

Reply:
By overriding up the `config.KeyLimit size in the benchmark function, the `watchchannel` buffer also needs to be initialised accordingly. Taken care of that as well.